### PR TITLE
skip json comment and allow trailing commas

### PR DESCRIPTION
**Pull request by xiaoxiaotank**
like this:
```json
{
  // it's a comment
  "HelloWorld": "Hello，World!",
}

```

### CODE REVIEW

**Comment by hishamco** [comment]
Could you please add a unit tests?

File: src/My.Extensions.Localization.Json/Internal/JsonResourceManager.cs [hunk 1]
```diff
@@ -178,9 +178,15 @@ IEnumerable<string> GetResourceFiles(string culture)
             var resources = new Dictionary<string, string>();
             if (File.Exists(filePath))
             {
-                var reader = new StreamReader(filePath);
+                var documentOptions = new JsonDocumentOptions
```
Comment (hishamco) [inline src/My.Extensions.Localization.Json/Internal/JsonResourceManager.cs:181]: It could be declared as a `private` member instead

**Review by hishamco — CHANGES_REQUESTED** [review]

**Comment by xiaoxiaotank** [comment]
Okay, I will re pr later

**Comment by hishamco** [comment]
Just modify this PR and push your commits

Thanks

**Comment by xiaoxiaotank** [comment]
I have submitted it, please check if it meets the requirements

**Review by hishamco — APPROVED** [review]

**Comment by hishamco** [comment]
Thanks for your contribution @xiaoxiaotank, hope to publish a release in upcoming days - if it's not today -

**Comment by xiaoxiaotank** [comment]
> 

You're welcome. I hope I can improve more functions in the future.

**Comment by hishamco** [comment]
I just checked the release seems I published `3.1.0` before your commits, I will see if there's another improvement I can add for `3.2.0`

**Pull request merged**